devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusGenerateCode.java

```diff
@@ -32,7 +32,7 @@
     public static final String QUARKUS_GENERATED_SOURCES = "quarkus-generated-sources";
     public static final String QUARKUS_TEST_GENERATED_SOURCES = "quarkus-test-generated-sources";
     // TODO dynamically load generation provider, or make them write code directly in quarkus-generated-sources
-    public static final String[] CODE_GENERATION_PROVIDER = new String[] { "grpc" };
+    public static final String[] CODE_GENERATION_PROVIDER = new String[] { "grpc", "avpr", "avsc" };
```

Comment:
it's really bad that we need this list. Can we access classpath resources in here?